### DISCUSSION TITLE
mspider/install/sidecar-update/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1182] [Announcements]
# mspider/install/sidecar-update/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/install/sidecar-update/

**Comment by mark8s** [comment]
热升级底层使用的是openkruise吗？

**Reply by kebe7jun** [reply]
不是的哦。

**Reply by mark8s** [reply]
@kebe7jun 您好，能不能透露下大致的底层实现呀

**Reply by kebe7jun** [reply]
主要基于 [Ephemeral Containers](https://kubernetes.io/docs/concepts/workloads/pods/ephemeral-containers/) 实现哈。
https://mp.weixin.qq.com/s/J9o9MNS7MJOaZNcvAEf-Wg

可以看这篇文章（有视频）了解哈。

**Comment by samzong** [comment]
配图需要更新
